R/RtGam.R

```diff
@@ -1,23 +1,90 @@
-#' Stub: Will become fitting/class constructor
+#' Fit a generalized additive model to incident cases
+#'
+#' # Model specification
+#' Incident cases are modeled as a smooth function of time with generalized
+#' additive models (GAMs). [RtGam] always fits a GAM, predicting incident
+#' cases as a smooth trend of time. However, the model adapts the penalty on
+#' wiggliness over time, allowing for changing epidemic dynamics.
```

Comment:
Is this only the case if there are >3 weeks of data? What about if there are <3 weeks of data?

Reply:
Clarified in documentation update.